autoload/lsp/ui/vim/output.vim

```diff
@@ -318,12 +322,10 @@ function! lsp#ui#vim#output#preview(server, data, options) abort
         let l:ft = a:options['filetype']
     endif
 
-    let l:server_info = lsp#get_server_info(a:server)
-    try
-        let l:do_conceal = l:server_info['config']['hover_conceal']
-    catch
-        let l:do_conceal = g:lsp_hover_conceal
-    endtry
+    let l:do_conceal = g:lsp_hover_conceal
+    let l:server_info = a:server != '' ? lsp#get_server_info(a:server) : {}
+    let l:config = get(l:server_info, 'config', {})
```

Comment:
@tyru want to fix?

Reply:
fixed.
(I don't know why vint does not output the error in my environment though...)